iwyu.cc

```diff
@@ -3941,10 +3941,9 @@ class IwyuAstConsumer
   // TODO(csilvers): we can probably relax this rule in .cc files.
   // TODO(csilvers): this should really move into IwyuBaseASTVisitor
   // (that way we'll correctly identify need for hash<> in hash_set).
-  // This is a Traverse*() because Visit*() can't call HandleFunctionCall().
-  bool TraverseTypedefDecl(clang::TypedefDecl* decl) {
-    if (!Base::TraverseTypedefDecl(decl))
-      return false;
+  // This is called from Traverse*() because Visit*()
+  // can't call HandleFunctionCall().
+  bool CheckAliasedClassMethods(TypedefNameDecl* decl) {
```

Comment:
I'd prefer `Handle...` to `Check...` here. "Check" implies "try-and-fail-hard" in IWYU, given the `CHECK_` macros.

Reply:
Thanks, fixed.

Reply:
I`m sorry, actually, isn't.

Reply:
Fixed at now.